code/modules/religion/rites.dm

```diff
@@ -218,3 +222,191 @@
+		to_chat(user, "<span class='warning'>Your target left the altar!</span>")
+		return FALSE
+	to_chat(user, "<span class='warning'>[moldify] becomes rancid!</span>")
+	user.emote("laughs")
+	new /obj/item/food/badrecipe/moldy(get_turf(religious_tool))
+	qdel(moldify)
+	return TRUE
+
+/datum/religion_rites/ritual_totem
+	name = "Create Ritual Totem"
+	desc = "Creates a Ritual Totem, a portable tool for performing rites on the go. Requires wood. Can only be picked up by the holy."
+	favor_cost = 100
+	invoke_msg = "Padala!!"
+	///the food that will be molded, only one per rite
+	var/obj/item/stack/sheet/mineral/wood/converted
+
+/datum/religion_rites/ritual_totem/perform_rite(mob/living/user, atom/religious_tool)
+	for(var/obj/item/stack/sheet/mineral/wood/could_totem in get_turf(religious_tool))
+		converted = could_totem //totemify this o great one
+		return ..()
+	to_chat(user, "<span class='warning'>You need at least 1 wood to do this!</span>")
+	return FALSE
+
+/datum/religion_rites/ritual_totem/invoke_effect(mob/living/user, atom/movable/religious_tool)
+	var/altar_turf = get_turf(religious_tool)
+	var/obj/item/stack/sheet/mineral/wood/padala = converted
+	converted = null
+	if(QDELETED(padala) || !(get_turf(religious_tool) == padala.loc)) //check if the same food is still there
+		to_chat(user, "<span class='warning'>Your target left the altar!</span>")
+		return FALSE
+	to_chat(user, "<span class='warning'>[padala] reshapes into a totem!</span>")
+	if(!padala.use(1))//use one wood
+		return
+	user.emote("laughs")
+	new /obj/item/ritual_totem(altar_turf)
+	return TRUE
+
+/obj/item/ritual_totem
+	name = "ritual totem"
+	desc = "A wooden totem with strange carvings on it."
+	icon_state = "ritual_totem"
+	inhand_icon_state = "sheet-wood"
+	lefthand_file = 'icons/mob/inhands/misc/sheets_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/misc/sheets_righthand.dmi'
+	force = 0
+	throwforce = 0
+
+/obj/item/ritual_totem/Initialize()
+	. = ..()
+	AddComponent(/datum/component/anti_magic, TRUE, TRUE, FALSE, null, 1, FALSE, CALLBACK(src, .proc/block_magic), CALLBACK(src, .proc/expire))//one charge of anti_magic
+	AddComponent(/datum/component/religious_tool, RELIGION_TOOL_INVOKE, FALSE)
+
+/obj/item/ritual_totem/proc/block_magic(mob/user, major)
+	if(major)
+		to_chat(user, "<span class='warning'>[src] consumes the magic within itself!</span>")
+
+/obj/item/ritual_totem/proc/expire(mob/user)
+	to_chat(user, "<span class='warning'>[src] quickly decays into rot!</span>")
+	qdel(src)
+	new /obj/effect/decal/cleanable/ash(drop_location())
+
+/obj/item/ritual_totem/can_be_pulled(user, grab_state, force)
+	. = ..()
+	return FALSE //no
+
+/obj/item/ritual_totem/examine(mob/user)
+	. = ..()
+	var/is_holy = user.mind?.holy_role
+	if(is_holy)
+		. += "<span class='notice'>[src] can only be moved by important followers of [GLOB.deity].</span>"
+
+/obj/item/ritual_totem/pickup(mob/taker)
```

Comment:
it's kind of weird that nobody but holy people can pick these up

maybe make it so that only holy people can get the antimagic benefit from it instead?